rst_files/julia_arrays.rst

```diff
@@ -268,41 +212,32 @@ In such cases, we don't care about the distinction between row and column vector
 This is why many Julia functions return flat arrays by default
 
 
-
-
-
 .. _creating_arrays:
 
 Creating Arrays
 ------------------
 
 
-
 Functions that Return Arrays
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 We've already seen some functions for creating arrays
 
 .. code-block:: julia
 
-    eye(2)
-
-
+    Matrix{Float64}(I, 2, 2)
```

Comment:
I think using `Matrix{Float64}` as a function would be confusing (especially since it takes 3 arguments). I think `ones(3, 3)` could work here as a different example.

Reply:
Is important to teach the current way of getting an identity matrix.
```
I # best case scenario
diagm(0 => ones(n))
Matrix{T}(I, n, n)
Diagonal(ones(n))
```
The `I` is covered, but we could choose ones of the other ones.